pkg/build/build.go

```diff
@@ -0,0 +1,85 @@
+package build
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+func Build(imageConfig *config.ImageConfig, buildConfig *config.BuildConfig) error {
+	err := prepareBuildDir(buildConfig)
+	if err != nil {
+		return err
+	}
+
+	err = ConfigureMessage(buildConfig)
+	if err != nil {
+		return err
+	}
+
+	err = cleanUpBuildDir(buildConfig)
+	return err
+}
+
+//go:embed scripts/script_base.sh
+var scriptBase string
+
+func prepareBuildDir(buildConfig *config.BuildConfig) error {
+
+	/* Combustion works by creating a volume with a subdirectory named "combustion"
```

Comment:
Comments describing the purpose of a function should instead be moved just above it. Usually multiple single-line comments ("//") are preferred to multi-line ones ("/*"). For reference: https://go.dev/doc/effective_go#commentary

Reply:
I'll change to using //, but I don't consider this documentation about the function itself since the caller doesn't need to know the details. The details are for the devs reading the function to give some idea of what the structure we're assembling looks like and why.